### PR TITLE
TASK: Replace curly braces syntax for string offset access

### PR DESCRIPTION
**Pull request by lorenzulrich**

### CODE REVIEW

**Review by Sebobo — APPROVED** [review]
Looks good by reading, but our CI doesn't seem to handle composer 2 correctly yet and the tests cannot run. We will have to fix that first.

File: Classes/Eel/IndexingHelper.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function buildAllPathPrefixes($path)
 
         $currentPath = '';
         $pathPrefixes = [];
-        if ($path{0} === '/') {
+        if (strpos($path, '/') === 0) {
```
Comment (kdambekalns) [inline Classes/Eel/IndexingHelper.php:53]: Why not just this?
```suggestion
        if ($path[0]) === '/') {
```

Comment (lorenzulrich) [reply Classes/Eel/IndexingHelper.php:53]: Sorry, I just used the same code from the 4.0 and master branches without doing actual thinking. Should I adjust or is it better to have it in sync?

Comment (kdambekalns) [reply Classes/Eel/IndexingHelper.php:53]: Ah, if it's like that let's keep it in sync! Sorry, I didn't do any further research myself… 🙈

**Review by kdambekalns — APPROVED** [review]

**Review by daniellienert — APPROVED** [review]

**Pull request merged**